src/main/kotlin/step4/view/ResultView.kt

```diff
@@ -0,0 +1,36 @@
+package step4.view
+
+import step4.model.Car
+
+object ResultView {
+    private const val CAR_NAME_SEPARATOR = " : "
+    private const val ONE_BLOCK = "-"
+
+    fun output(numberOfTrials: Int, cars: List<Car>) {
+        for (i in 0 until numberOfTrials) {
+            val actualTrial = i + 1
+
+            println("trial$CAR_NAME_SEPARATOR$actualTrial")
+            outputForTrials(actualTrial, cars)
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        (1..numberOfTrials).forEach {
            println("trial$CAR_NAME_SEPARATOR$it")
            outputForTrials(it, cars)
        }
```

Reply:
반영하였습니다.